platform/lib/schema/index.ts

```diff
@@ -575,6 +575,75 @@ export function mapOf<K, V>(
   return new MapOfSetting(keySetting, valueSetting, options);
 }
 
+export function oneOf<A, B, C, D, E, F, G, H, I, J>(
```

Comment:
`oneOf` just needed to handle more parameters, as we used it for a lot of `literal`s in the logging code.